Makefile

```diff
@@ -351,6 +351,18 @@ build: bin-dir require-jq require-go require-git verify-go-versions  ## Build th
 .PHONY: bin-dir
 bin-dir:
 	mkdir -p bin
+##@ Testing
+
+.PHONY: test-e2e-argocd
+test-e2e-argocd: ## Run ArgoCD integration E2E tests
+	@echo "🚀 Running ArgoCD E2E tests..."
+	cd test/e2e && ./run-test.sh --test-type argocd-ginkgo --env kind
```

Comment:
Is it really necessary to do this `cd` first? For the other E2E tests, it is _not_ necessary to do that `cd` first; for example, the test can be invoked from the repo's root directory (`test/e2e/run_test.sh ...`). I recommend keeping that true for the new test suite too.